### DISCUSSION TITLE
Energy - it would be useful to have a configuration

### DISCUSSION THREAD

**Discussion by Santanachia** [home-assistant/frontend#14083] [Dashboard]
It would be great to be able to configure the dashbord `Energy`. At least be able to show/hide/move cards.

**Comment by matthiasdebaat** [comment]
Can you give some more info about what you want to configure?

**Comment by Santanachia** [comment]
I managed to find where the error is :D

in the query there is no column I want to retrieve.

The error is not in the connection, but in the error message.
![image](https://user-images.githubusercontent.com/4628507/195602464-d2007cc3-ffb7-4736-bbee-f46b9d092b88.png)
